frontend/packages/client/src/hooks/useCommunity.js

```diff
@@ -99,8 +99,9 @@ export default function useCommunity({
           websiteUrl,
           logo,
           banner,
-          contractAdrress: contractAddr,
+          contractAddress: contractAddr,
```

Comment:
good catch! (adding typescript in the future would be awesome!)

Reply:
Totally! That actually cost me a bit of time because I couldn't figure out why the contract was failing.